src/lib/stitching/exchange/v2/stitching.ts

```diff
@@ -185,6 +185,55 @@ export const exchangeStitchingEnvironment = ({
     },
   }
 
+  const paymentMethodResolver = {
+    fragment: gql`
+      fragment CommerceOrderPaymentMethod on CommerceOrder {
+        creditCardId
+        bankAccountId
+        paymentMethod
+      }
+    `,
+
+    resolve: async (parent, _args, context, info) => {
+      const { creditCardId, bankAccountId, paymentMethod } = parent
+
+      if (paymentMethod === "credit card") {
+        // assert creditCardId exists?
+        return info.mergeInfo.delegateToSchema({
+          schema: localSchema,
+          operation: "query",
+          fieldName: "creditCard",
+          args: {
+            id: creditCardId,
+          },
+          context,
+          info,
+          transforms: exchangeSchema.transforms,
+        })
+        // note: us_bank_account with underscore *is* the current value from exchange
+      } else if (paymentMethod === "us_bank_account") {
+        // assert bankAccountId exists?
+        return info.mergeInfo.delegateToSchema({
+          schema: localSchema,
+          operation: "query",
+          fieldName: "bankAccount",
+          args: {
+            id: bankAccountId,
+          },
+          context,
+          info,
+          transforms: exchangeSchema.transforms,
+        })
+      } else if (paymentMethod === "wire transfer") {
+        return {
+          // PaymentMethodUnionType relies on this property
+          // not sure if this is the best way to model a manual payment TBH
+          _isManualPayment: true,
```

Comment:
Why the underscore?

Reply:
my logic in the moment: in this file i am returning an object that will be used to identify the type as a manual wire transfer. in the other file i am using this internal property to identify the type, which happens to have a single field of the same name. Yes, i agree it doesn't make sense.

Reply:
I know it seems silly, but maybe do `__isManualPayment`. `_foo` -- one underscore -- means "private variable", where's the double underscore (`__whatever`) is used for marking internal / otherwise clashing implementation details